File: InfiniteScrollViewDemo/PhotoViewController.m
//
//  PhotoViewController.m
//  InfiniteScrollViewDemo
//
//  Created by pronebird on 4/4/15.
//  Copyright (c) 2015 codeispoetry.ru. All rights reserved.
//

#import "PhotoViewController.h"

@implementation PhotoViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.imageView.image = self.photo;
}

- (IBAction)dismiss:(id)sender {
    [self dismissViewControllerAnimated:YES completion:nil];
}

@end

File: InfiniteScrollViewDemoSwift/PhotoViewController.swift
//
//  PhotoViewController.swift
//  InfiniteScrollViewDemoSwift
//
//  Created by pronebird on 5/3/15.
//  Copyright (c) 2015 pronebird. All rights reserved.
//

import UIKit

class PhotoViewController: UIViewController {
    
    @IBOutlet weak var imageView: UIImageView?
    var photo: UIImage? {
        didSet {
            imageView?.image = photo
        }
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
    
        imageView?.image = photo
    }
    
    @IBAction func dismiss(_ sender: AnyObject!) {
        self.dismiss(animated: true, completion: nil)
    }
}
